app/src/main/java/com/wizeline/compose/academy/screens/home/HomeScreen.kt

```diff
@@ -0,0 +1,107 @@
+package com.wizeline.compose.academy.screens
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Notifications
+import androidx.compose.material.icons.filled.Search
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.wizeline.compose.academy.R
+import com.wizeline.compose.academy.components.*
+import com.wizeline.compose.academy.ui.theme.ComposeAcademyTheme
+import com.wizeline.compose.academy.ui.theme.PrimaryGravyVariant
+import com.wizeline.compose.academy.ui.theme.PrimaryGray
+
+
+@Composable
+fun HomeScreen() {
+    Column(modifier = Modifier.fillMaxSize().padding(20.dp)) {
```

Comment:
Here if you're going to use this modifier in all your screens you can do this

```
fun Modifier.defaultScreen() = this.fillMaxSize().padding(20.dp)
```

 And now you can use this function like 

```
Modifier.defaultScreen()
```

Reply:
thank you very much for your comments :D